### PR TITLE
Trackstore fixes & more logging

### LINKED ISSUE #652: Cannot start powergate docker
2020-10-05T05:10:44.410Z	INFO	powd	powd/main.go:68	starting server...
2020-10-05T05:10:44.419Z	INFO	server	server/server.go:177	Detected Lotus node connected to network: ignition
2020-10-05T05:10:45.911Z	INFO	fchost	fchost/fchost.go:116	connected to 6 out of 6 bootstrap peers
2020-10-05T05:10:45.912Z	INFO	fchost	fchost/fchost.go:60	bootstraping libp2p host dht
2020-10-05T05:10:45.912Z	INFO	fchost	fchost/fchost.go:64	dht bootstraped!
2020-10-05T05:10:45.912Z	INFO	server	server/server.go:555	Opening badger database...
2020-10-05T05:10:46.049Z	INFO	server	server/server.go:194	Wiring internal components...
2020-10-05T05:10:46.051Z	INFO	index-ask	runner/runner.go:65	loaded persisted index with 283 entries
2020-10-05T05:10:46.108Z	INFO	index-miner	module/meta.go:44	updating meta index...
2020-10-05T05:10:46.111Z	INFO	index-miner	module/onchain.go:36	updating on-chain index...
2020-10-05T05:10:46.114Z	INFO	reputation	reputation/reputation.go:200	rebuilding index
2020-10-05T05:10:46.114Z	INFO	reputation	reputation/reputation.go:227	built scores for 283 miners in 0ms
2020-10-05T05:10:46.186Z	FATAL	powd	powd/main.go:71	starting server: creating scheduler: loading scheduler trackstore: loading renewable/repairable caches: decoding cid: selected encoding not supported

### PR DESCRIPTION
**Pull request by jsign**
- Closes #652 (confirmed by user in chat)
- Adds some extra logging to debug a weird situation. The user hasn't replied to test this, but since it's only logging shouldn't care much for merging.

### CODE REVIEW

File: api/server/server.go [hunk 1]
```diff
@@ -153,16 +153,9 @@ func NewServer(conf Config) (*Server, error) {
...
-	if conf.Devnet {
-		// Wait for the devnet to bootstrap completely and generate at least 1 block.
-		time.Sleep(time.Second * 6)
-		if masterAddr, err = c.WalletDefaultAddress(context.Background()); err != nil {
-			return nil, fmt.Errorf("getting default wallet addr as masteraddr: %s", err)
-		}
-	} else {
-		if masterAddr, err = address.NewFromString(conf.LotusMasterAddr); err != nil {
-			return nil, fmt.Errorf("parsing masteraddr: %s", err)
-		}
```
Comment (jsign) [inline api/server/server.go:165]: Just extracting logic.

File: ffs/scheduler/internal/trackstore/trackstore.go [hunk 2]
```diff
@@ -107,6 +108,9 @@ func (s *Store) Remove(c cid.Cid) error {
 	if err := s.ds.Delete(datastore.NewKey(c.String())); err != nil {
 		return fmt.Errorf("deleting a tracked storage config: %s", err)
 	}
+	delete(s.renewables, c)
+	delete(s.repairables, c)
```
Comment (jsign) [inline ffs/scheduler/internal/trackstore/trackstore.go:112]: Fixes.

File: ffs/scheduler/internal/trackstore/trackstore.go [hunk 3]
```diff
@@ -152,7 +156,8 @@ func (s *Store) loadCaches() error {
 		if err := json.Unmarshal(v.Value, &tsc); err != nil {
 			return fmt.Errorf("unmarshaling storageconfig: %s", err)
 		}
-		c, err := cid.Decode(v.Key)
+		cidStr := strings.TrimPrefix(v.Key, "/")
+		c, err := cid.Decode(cidStr)
```
Comment (jsign) [inline ffs/scheduler/internal/trackstore/trackstore.go:160]: Fix.

File: ffs/scheduler/internal/trackstore/trackstore_test.go [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"github.com/textileio/powergate/ffs"
+	"github.com/textileio/powergate/tests"
+	"github.com/textileio/powergate/util"
+)
+
+func TestAll(t *testing.T) {
```
Comment (jsign) [inline ffs/scheduler/internal/trackstore/trackstore_test.go:12]: More tests.

File: index/miner/module/miner.go [hunk 5]
```diff
@@ -201,19 +202,20 @@ func (mi *Index) loadFromDS() error {
...
 	buf, err := mi.ds.Get(dsKeyMetaIndex)
 	if err != nil && err != datastore.ErrNotFound {
-		return err
+		return fmt.Errorf("getting metadata key value: %s", err)
 	}
 	if err == nil {
 		var metaIndex miner.MetaIndex
 		if err := json.Unmarshal(buf, &metaIndex); err != nil {
-			return err
+			log.Warnf("wrong json?: %s", string(buf))
```
Comment (jsign) [inline index/miner/module/miner.go:210]: Wanting user to see how this log is on it's weird situation.

**Review by asutula — APPROVED** [review]

**Pull request merged**